### DISCUSSION TITLE
Changes are not being reflected on reload. Next.js 12.0.8 Tailwind 3.0.15

### DISCUSSION THREAD

**Discussion by idavidgeo** [tailwindlabs/tailwindcss#7189] [Help] [answered]
Long time user of tailwind, never had this issue. Issue only came from starting a fresh NextJS/Tailwind project.

Important to note: The refresh works fine on the first 1-2 changes, after that it doesn't.
The server still shows "wait  - compiling..." - like usual
The client still shows "[Fast Refresh] rebuilding" - like usual

But no visual changes in the browser. Tested on Chrome, Safari, Firefox, running on MacOS.

I can upload my repo if you can't reproduce.

Here's my package.json :
```
{
  "name": "mysite-test",
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint"
  },
  "dependencies": {
    "@metamask/eth-sig-util": "^4.0.0",
    "@prisma/client": "^3.8.1",
    "axios": "^0.25.0",
    "cookie": "^0.4.1",
    "ethereumjs-util": "^6.2.1",
    "jsonwebtoken": "^8.5.1",
    "next": "^12.0.8",
    "react": "17.0.2",
    "react-dom": "17.0.2",
    "sqlite": "^4.0.23",
    "uuid": "^8.3.2",
    "valtio": "^1.2.9",
    "web3": "^1.7.0"
  },
  "devDependencies": {
    "autoprefixer": "^10.4.2",
    "eslint": "8.7.0",
    "eslint-config-next": "12.0.8",
    "postcss": "^8.4.5",
    "prisma": "^3.8.1",
    "tailwindcss": "^3.0.15"
  }
}
```

**Comment by idavidgeo** [comment, marked as answer]
Never-mind, I found the issue. The issue was only observed in components inside the components folder. The issue was that the component name was nav.js, but was imported as components/Nav in pages.

**Reply by maorazoulay** [reply]
Wow you saved my project, thank you so much, I can't believe VSCode is too dumb to give us an error or warning about this.
